src/pages/resultsView/comparison/ResultsViewComparisonStore.ts

```diff
@@ -171,6 +171,7 @@ export default class ResultsViewComparisonStore extends ComparisonStore {
     protected async saveAndGoToSession(newSession: Session) {
         const { id } = await comparisonClient.addComparisonSession(newSession);
         this.urlWrapper.updateURL({ comparison_createdGroupsSessionId: id });
+        this.newSessionPending = false;
```

Comment:
what's reason for this?

Reply:
newSessionPending is an observable on the comparison store which is set whenever an action is initiated that involves creating a new comparison session (such as creating or deleting a group). its used for displaying a loading indicator. at this point, at the end of `saveAndGoToSession`, the new session is surely no longer pending, so it is set to false. this fixes an issue where the loading indicator was never disappearing - we didn't see that issue before because the whole page remounted (which it needn't do) whenever a new group was added or deleted from the comparison tab.